Makefile

```diff
@@ -570,7 +570,7 @@ PACK_ARGS=\
 	RAMP_YAML=$(RAMP_YAML) \
 	RAMP_ARGS=$(RAMP_ARGS)
 
-RAMP.release:=$(shell JUST_PRINT=1 RAMP=1 DEPS=0 RELEASE=1 SNAPSHOT=0 $(PACK_ARGS) $(ROOT)/sbin/pack.sh)
```

Comment:
We don't have `DEPS` leftovers, right?

Reply:
Only in makefile, something regarding libuv, https://github.com/RediSearch/RediSearch/blob/eca584a11aef6cdb17bf2524ffbba374961e7c30/Makefile#L343